game_patch/multi/server.cpp

```diff
@@ -42,9 +43,16 @@ const char* g_rcon_cmd_whitelist[] = {
     "map_prev",
 };
 
+std::vector<std::shared_ptr<RespawnPoint>> respawn_points;
```

Comment:
I do not understand. `g_multi_respawn_points` is at 0x006C6C40.

Reply:
I made an intentional decision not to use the existing array for a few reasons:
1. It's easier to work with this way
2. I intend to add additional functionality to this - like allowing mappers to assign weights to specific spawn points, and it's needed in that case
3. I wanted to support >32 spawn points in a single map... which I am now realizing isn't working properly and I need to fix. Moving this PR to draft while I fix that